### PR TITLE
[NetworkExtension] Add Xcode 13 beta 4 support.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

File: tests/xtro-sharpie/MacCatalyst-NetworkExtension.ignore [hunk 1]
```diff
@@ -0,0 +1 @@
+!duplicate-type-name! NWPathStatus enum exists as both NetworkExtension.NWPathStatus and Network.NWPathStatus
```
Comment (spouliot) [inline tests/xtro-sharpie/MacCatalyst-NetworkExtension.ignore:1]: weird, that type has not changed in the PR

**Review by spouliot — APPROVED** [review]

File: tests/xtro-sharpie/MacCatalyst-NetworkExtension.ignore (hunk 1 above)
Comment (mandel-macaque) [reply tests/xtro-sharpie/MacCatalyst-NetworkExtension.ignore:1]: Was not present in catalyst, was in macOS. And the ignore of that type is in the macOS ignore, that's why we had to add it for catalyst

Comment (spouliot) [reply tests/xtro-sharpie/MacCatalyst-NetworkExtension.ignore:1]: yeah - but you did not change `src/frameworks.sources` (or any other files) in that PR, so
if it was present before, then it should have been an error in xtro
if it was not, then it was not added, and should not be reported

**Comment by mandel-macaque** [comment]
Unrelated error: https://github.com/xamarin/maccore/issues/2414

**Pull request merged**